### ISSUE TITLE
mt7610u driver compilation & installation issue

### ISSUE THREAD

**Issue by simdol** [ulli-kroll/mt7610u#37]
Hey all,

I am having issues with compilation of the driver (undefined functions), and I am unable to install the kernel module.

Build: https://nopaste.me/view/5c79bfc0 [GCC 6.4.0]
dmesg: https://nopaste.me/view/cfe0e02a#L29

Please note that I've compiled kernel on my own, with cfg80211, and mac80211 built into the kernel. 

Thanks!

**Comment by ulli-kroll** [comment]
> Please note that I've compiled kernel on my own, with cfg80211, and mac80211 built into the kernel.
is
`/usr/src/pf-kernel`
your kernel source directory ??
Is your build setup complete ??
because most of the missing symbols in
https://nopaste.me/view/cfe0e02a#L29
are essential symbols and used by any driver
i.e.
up(), down()

**Comment by simdol** [comment]
Yes, that is my kernel source directory. The build setup tools are complete, since I pretty much completed everything.

What exactly am I missing here?

**Comment by ulli-kroll** [comment]
> What exactly am I missing here?

I don't know ..
Unless 
`/usr/src/pf-kernel`
is your $host kernel and your .config is complete 
maybe there is
`make modules_prepare`
needed.

I do frequently several build on $host and $target with cross toolchain
